lib/services.identity.js

```diff
@@ -669,37 +669,9 @@ function updateIdentity(req, res, next) {
       brIdentity.setRoles(req.user.identity, identity, callback);
     }],
     update: ['updateRoles', function(callback, results) {
-      var identity = {id: results.getId};
-
-      // conditional values only set if preset
-      if(req.body.description) {
-        identity.description = req.body.description;
-      }
-      if(req.body.email) {
```

Comment:
Email field is not in https://github.com/digitalbazaar/bedrock-identity/pull/11. Should it be?